packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -99,66 +105,74 @@ class BaseStepBuilder {
   }
 
   // STEPS
-  createRow(inputs: CreateRowStepInputs): this {
+  createRow(inputs: CreateRowStepInputs, stepName?: string): this {
```

Comment:
Should the step name be within an object specifier? E.g. `opts?: { name?: string }` - that way if there are further optional parameters in future they are easy to add without needing to update everywhere these are used.

Reply:
Sorted 👍